src/index.js

```diff
@@ -9,7 +9,16 @@ const ReaperBase = require('./reaper-base')
  * @returns {(import('./reaper-base').ReaData|ReaperBase)}
  */
 function parse (rppString) {
-  return parser.parse(rppString)
+  try {
+    return parser.parse(rppString)
+  } catch (err) {
+    const {start, end} = err.location,
+      lines = rppString.split(/\r?\n/ig).map(l => '    ' + l)
+    err.nearLines = lines.slice(start.line - 4, start.line - 1)
```

Comment:
Thanks for the PR, @JohnnyMarnell ! I'm publishing v0.4.2 to npm, which includes JSFX support. 

I'm thinking about adding this too, but can you confirm that this works as expected when the calculated indices are outside the bounds of the array? For example when the error occurs on the very first or last line in the input string?